presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -45,6 +46,11 @@ public HiveSessionProperties(HiveClientConfig config)
     {
         sessionProperties = ImmutableList.of(
                 booleanSessionProperty(
+                        BUCKET_EXECUTION_ENABLED,
+                        "Assign one bucket to each worker",
```

Comment:
This isn't really correct, since the number of buckets and workers aren't always equal. For example, a worker will be assigned more than one bucket if there are more buckets than workers.

We could say "Enable bucket-aware execution: only use a single worker per bucket"